Dockerfile

```diff
@@ -57,3 +59,5 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
     org.label-schema.vcs-ref=$VCS_REF \
     org.label-schema.vcs-url="https://github.com/aquasecurity/kube-bench" \
     org.label-schema.schema-version="1.0"
+
+COPY helper_scripts/check_files_owner_in_dir.sh /go/bin
```

Comment:
better to have this step before we define entrypoint for better readability. we can put next to other copy commands